src/layouts/_ContentLayout.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import MainContainer from '@cerner/terra-application/lib/main-container';
+import Suspense from '@cerner/terra-application/lib/shared/Suspense';
+import classNamesBind from 'classnames/bind';
+import ContentLoadedContainer from '../content/_ContentLoaded';
+import { contentImportsShape, pageContentConfigShape } from '../site/siteConfigShapes';
+
+import styles from './ContentLayout.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
+  /**
+   * Config describing the page
+   */
+  pageContentConfig: pageContentConfigShape.isRequired,
+
+  /**
+   * Function called to request closing the modal
+   */
+  contentImports: contentImportsShape.isRequired,
+};
+
+const ContentLayout = ({ pageContentConfig, contentImports }) => {
+  const location = useLocation();
+  const { pathname } = location;
+  const ContentComponent = contentImports[pathname];
+  const [loadingFailed, setLoadingFailed] = React.useState();
+
+  if (!pageContentConfig) {
+    return <div>404</div>;
```

Comment:
![404](https://user-images.githubusercontent.com/6720991/114239719-0b3c5680-994c-11eb-85a5-c9429f5dae67.gif)

Reply:
![https://media.giphy.com/media/NEvPzZ8bd1V4Y/giphy.gif]

Reply:
I'll make this update in a future PR.